### PR TITLE
Fixed Issue#3080 Removed the dropdown if User has only links.

### LINKED ISSUE #3080: [FEATURE] If profile only has links then do not show dropdown or tab
### Description

Currently, a dropdown is rendered on mobile regardless of if it is needed or not.  This issue is to handle not rendering a dropdown for profiles that contain only links and only render a dropdown on 2 or more items.

### Screenshots

Current mobile view with links only, dropdown has been clicked in this image. Dropdown is not needed here
Screenshot Amanda profile on android with dropdown clicked showing it has one item only

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by dhruvilmehta**
## Fixes Issue
Dropdown removed if the profile has only personal links to show.

## Changes proposed
The design of the MyLinks option remains same in the mobile view as that of the laptop view (just comes in the middle of the screen) if the user has only MyLinks. The dropdown is only shown if the user has other options like Milestones, Testimonials, etc

## Check List (Check all the applicable boxes) 

- [x ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x] All new and existing tests passed.
- [ x] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.

## Screenshots
Screenshot 2023-04-03 155721
Example: User has only MyLinks.
Screenshot 2023-04-03 160056
Example: If User has more than 1 option.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5816"></a>

### CODE REVIEW

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]
You have pushed your profile update in this PR.  Can you please remove that and place in a separate PR as it is not related to the changes required for this issue. Thanks!

**Comment by dhruvilmehta** [comment]
Removed my profile and made another commit to Issue#3080 branch.

File: components/user/UserTabs.js [hunk 1]
```diff
@@ -30,6 +31,10 @@ export default function UserTabs({ tabs, setTabs }) {
             <option key={tab.name}>{tab.name}</option>
           ))}
         </select>
+        }
+        {tabs.length==1 && 
+          <div class="border-b border-gray-200"><nav class="-mb-px flex" style={{justifyContent: "center"}}aria-label="Tabs"><Link rel="noreferrer" class="border-indigo-500 text-indigo-600 w-1/4 py-4 px-1 text-center border-b-2 font-medium text-sm" aria-current="page" href="/dhruvilmehta#">My Links ({tabs[0].total})</Link></nav></div>
```
Comment (eddiejaoude) [inline components/user/UserTabs.js:36]: not sure why this has your hardcoded user name in it? I think the extra nav is not required

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for your contribution. Inline comment left.

Please also refer to the issue in the PR description

File: components/user/UserTabs.js (hunk 1 above)
Comment (dhruvilmehta) [reply components/user/UserTabs.js:36]: Yes, very sorry for that mistake.

**Comment by dhruvilmehta** [comment]
Do you want me to add inline comment for my code ?

File: components/user/UserTabs.js [hunk 2]
```diff
@@ -12,13 +12,14 @@ export default function UserTabs({ tabs, setTabs }) {
...
   };
-
+  
   return (
     <div>
       <div className="sm:hidden">
         <label htmlFor="tabs" className="sr-only">
           Select a tab
         </label>
+        {tabs.length>1 &&
```
[the remark this answers is not in the document]
Comment (dhruvilmehta) [reply components/user/UserTabs.js:22]: I have not changed any of the styling. I am using VScode only.

**Comment by dhruvilmehta** [comment]
Done necessary changes and added inline comments.

File: components/user/UserTabs.js [hunk 3]
```diff
@@ -30,6 +31,13 @@ export default function UserTabs({ tabs, setTabs }) {
             <option key={tab.name}>{tab.name}</option>
           ))}
         </select>
+        }
+        {tabs.length==1 && // if tabs has only links in it, then no dropdown is needed
```
Comment (eddiejaoude) [inline components/user/UserTabs.js:35]: I don't think this section is required, nothing needs to be shown if only links are available 

Could you add the issue link to the PR description please

File: components/user/UserTabs.js [hunk 4]
```diff
@@ -12,13 +12,14 @@ export default function UserTabs({ tabs, setTabs }) {
...
   };
-
+  
   return (
     <div>
       <div className="sm:hidden">
         <label htmlFor="tabs" className="sr-only">
           Select a tab
         </label>
+        {tabs.length>1 && // if tabs contains more than one options, then show dropdown menu
```
Comment (eddiejaoude) [inline components/user/UserTabs.js:22]: I think the code is self explanatory, but I think the formatting is not the correct coding style, VScode should correct this, you might need to install the Prettier plugin

**Review by eddiejaoude — APPROVED** [review]
Thank you for your contribution 👍 

I will merge and tweak with a temporary branch

**Comment by dhruvilmehta** [comment]
I see that you have removed tabs.length==1 part, but I think that part is required. After removing that code, if tabs.length==1, then it will show nothing, but we need to show My links option right ? As you can see in the first image which I uploaded, if tabs.length==1, then it shows MyLinks. That part is of code which you removed.

**Comment by eddiejaoude** [comment]
After seeing the "my links" I felt it made more sense to remove displaying anything if there was no other options

**Comment by dhruvilmehta** [comment]
Ok, No problem.

**Pull request merged**